Desktop version

iambenkis opened the pull request:
# Checklist
- [ ] My code follows the style guidelines of this project
- [ ] My portfolio display correctly in the desktop version according to the Figma template
- [ ] I have commented my code, particularly in hard-to-understand areas 
- [ ] My changes generate no new warnings

iambenkis committed:
add home page of desktop

iambenkis committed:
add project section on desktop version

iambenkis committed:
update contact section in desktop version

iambenkis committed:
update flex-direction with row-reverse on project

iambenkis committed:
update Readme file

iambenkis committed:
solve linter error

tresorsawasawa commented on index.html:
```diff
@@ -19,11 +19,22 @@
         <nav class="nav-bar"> 
             <a href="#" class="logo-txt">My Logo</a> 
             <i class="fa-solid fa-bars"></i>
+            <ul class="nav-list">
+                <li>
+                    <a href="#projects-container">Portfolio</a>
+                </li>
+                <li>
+                    <a href="#about">About</a>
+                </li>
+                <li>
+                    <a href="#contact">Contact</a>
+                </li>
+            </ul>
```
- [ **OPTIONAL** ] I think it would be better if you add on all the navigation items a `cursor pointer` property to help users immediately know that they are all clickable.

tresorsawasawa commented on style.css:
```diff
@@ -351,3 +364,181 @@ textarea {
 }
+
+@media (min-width: 600px) {
+  .mobile {
```
- [ ] Good job with your styling. But according to the project requirements you are supposed to use media queries for the Desktop breakpoint with the value for the `min-width` property to be in the range of `768px` or wider. According to your page, I suggest you use the ``min-width`` property of `920px` or another value not less than `900px`.

tresorsawasawa commented on style.css:
```diff
@@ -351,3 +364,181 @@ textarea {
+    justify-content: space-between;
+    width: 20%;
+  }
+
```
- [ ] Kindly make sure you use the correct spacing value between icon items(in the **home section**) (i.e.: `.icons li`). I suggest you check the correct value used from the Figma template and apply it to your project. I noticed that when a user is trying to change screen sizes they(icons) seem to overlap each other.

- [ **OPTIONAL** ] What do you think if you add on all the icons items a `cursor pointer` property to help users immediately know that they are all clickable and as they are all links.

tresorsawasawa commented on style.css:
```diff
@@ -351,3 +364,181 @@ textarea {
+    justify-content: space-between;
+  }
+
+  .about-text {
+    width: 50%;
+    margin-right: 7rem;
+  }
+
+  .knowledge {
+    width: 40%;
+  }
+
```
- [ ] Kindly make sure you apply the correct width value on the `About-text` block and `knowledge` block. If you correctly check on the Figma template you will realize that the `About-text`  block has a width less than the one of the `knowledge`(e.g.:  `.about-text { width: 40%; }` and `.knowledge { width: 60%; }`). Make sure you fix this issue to stick to the Figma design.

tresorsawasawa reviewed (changes requested):
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @iambenkis 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

![almost-there](https://user-images.githubusercontent.com/57562869/143511358-9cde7daf-4ad0-4db2-8f15-18b54288ff91.gif)

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

iambenkis commented:
# Checklist Of review 
- [ ] My code follows the style guidelines of this project
- [ ] No Linter  error.
- [ ] I added cursor with pointer value so allow good navigation in the web page
- [ ] I changed the value of the ```min-width``` to a value between ```768px```  and wider.
- [ ] I change the width of the ```knowledge```  and  ```About-text``` classes.

iambenkis committed:
apply changes requested

iambenkis committed:
correct stylelint

gtekle commented on index.html:
```diff
@@ -45,8 +56,105 @@ <h2 class="connect-txt">LET'S CONNECT</h2>
-    <section class="projects-container">
-        <div class="project">
+    <section class="projects-container" id="projects-container"> 
+        <div class="desktop project">
+            <img src="./images/project5.png" class="project-img" alt="Tonic Image">
+            <div class="project-txt">
+                <h2 class="project-title">Tonic </h2>
+                <div class="caption">
+                    <p class="canopy">CANOPY</p>
+                    <img src="./images/Counter.png" alt="">
+                    <p>Back end dev</p>
+                    <img src="./images/Counter.png" alt="">
+                    <p>2015</p>
+                </div>
+                <p class="text">
```
- [ Optional] You should not have duplicate content for the sole purpose of rendering them in different screen sizes. The HTML content remains the same and the only need you need to add is CSS styling using media queries.
![image](https://user-images.githubusercontent.com/44057863/177863787-098a3327-13e4-4ff0-8ee6-b97aa33176e2.png)
You may refer this [article](https://www.w3schools.com/cssref/sel_nth-child.asp) on how to use an `nth child` css property and use FlexBox `order` attribute to dynamically swap the place of the project images in the desktop version.

gtekle reviewed (approved):
Hi @iambenkis :wave: , 

# STATUS: APPROVED :tada: :tada: :tada: :100: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
![approved-stamp](https://user-images.githubusercontent.com/44057863/177865469-b61a6716-346e-4803-b3cb-705081cbbd0d.gif)


## Highlights
:heavy_check_mark: Nice UI
:heavy_check_mark: Good use of Github flow
:heavy_check_mark: Descriptive PR summary
:heavy_check_mark: Meaningful commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
